src/app/core/app.component.ts

```diff
@@ -28,6 +28,7 @@ export class AppComponent implements OnInit, OnDestroy {
   );
 
   leftMenuAndHeaders$ = this.layoutService.leftMenuAndHeadersDisplayed$;
+  withTask$ = this.layoutService.withTask$;
```

Comment:
Don't you want the same for course? If so, using "task" in the name is not very appropriate.

Reply:
TODO : Rename leftMenuAndHeaders to "fullFrameContent", "withTask" to "footer"

Reply:
Renamed.